src/embed.js

```diff
@@ -28,18 +28,17 @@ matchers.push(spotify);
  * Adds title attribute or title span element if missing from embed html
  *
  * @param {string} html html of the embed
- * @returns {string} returns either iframe with title attribute or html with a span element containing a title
+ * @returns {string} returns either iframe with title attribute or html with a span with a title
  */
 function addTitle(html, title) {
   const dom = new JSDOM(html);
   const doc = dom.window.document;
   const iframe = doc.getElementsByTagName('iframe');
 
-  if (iframe.length !== 0 && !(iframe.item(0).title)){
-    const body = iframe.item(0).title = title;
-  }
-  else{
-    const span = doc.createElement("span");
+  if (iframe.length !== 0 && !(iframe.item(0).title)) {
+    iframe.item(0).title = title;
+  } else {
+    const span = doc.createElement('span');
```

Comment:
Isn't it sufficient to add a `title` attribute? Why do we have to duplicate the content in a span.

Reply:
@trieloff I did this because the code already has this baked in and this is how we seem to prefer to include a title in an embeds html. 

https://github.com/adobe/helix-embed/blob/85b555b356438c88932a7f12c6be1a13ce03b235/src/embed.js#L65-L72

Reply:
Ok. I wonder what I've been thinking. But we definitely must add the `title` attribute to the `iframe`.

Reply:
also i think trying to put the attribute in the right place can become a little weird; should we put it where the <script ...> or should we find another element below that like lottieplayer and stick a title attribute in there? That placement would be pretty complex

Reply:
@trieloff no problem that is done! IF we can't find a title we just put "content from <<embed source>>". should I leave the span code?

Reply:
Yes, leave the span code, as taking it away would be a breaking change.